features/command_line/fail_fast.feature

```diff
@@ -3,17 +3,29 @@ Feature: `--fail-fast` option
   Use the `--fail-fast` option to tell RSpec to stop running the test suite on
   the first failed test.
 
-  You may also specify `--no-fail-fast` to turn it off (default behaviour).
+  You may add a parameter to tell RSpec to stop running the test suite after N
+  failed tests, for example: `--fail-fast=3`.
```

Comment:
I won't be able to concatenate any other command if I drop the `=`. Maybe I am doing something wrong but if I drop it, it will see anything following `fail-fast` as a parameter for `fail-fast` itself, therefore if I add another command it will error. 

For example we have `--bisect` that requires an `=`.
What do you suggest?